anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -170,29 +166,38 @@ let handleExport
     | Entities -> $"distinct %s{aidColumn}"
 
   let countColumn = $"diffix_count(%s{countInput}, %s{aidColumn}) AS count"
+  let columns = Array.concat [ buckets; [| countColumn |] ]
+  let countColumnIndex = Array.findIndex (fun column -> column = countColumn) columns
+
+  let bucketColumnGroupByIndices =
+    columns
+    |> Array.indexed
+    |> Array.filter (fun (_, column) -> column <> countColumn)
+    // GROUP BY expects 1-based indexing
+    |> Array.map (fst >> (+) 1)
```

Comment:
I don't understand why this part is needed at all. The `SELECT` clause is manually assembled right here, so we know that the `count` column is always the last one. The old version of the code seems fine to me.

Reply:
OK, I thought that it was easier to mix things up that way, but I wasn't sure, I can take it all back and just fix the bug surgically.